source/rust.txt

```diff
@@ -37,35 +37,78 @@ See `Installation <https://github.com/mongodb/mongo-rust-driver#Installation>`__
 Connect to MongoDB Atlas
 ------------------------
 
+Select from the :guilabel:`Sync` or :guilabel:`Async` tabs below for
+corresponding connection code samples.
+
 .. include:: /includes/atlas-connect-blurb.rst
 
-.. code-block:: rust
+.. tabs::
+
+   .. tab:: Sync
+      :tabid: rust-sync
+
+      Make sure you enabled the sync API. See
+      `Enabling the sync API <https://github.com/mongodb/mongo-rust-driver#enabling-the-sync-api>`__
+      for more details.
+
+      .. code-block:: rust
+
+         use mongodb::{bson::doc, sync::Client};
+
+         fn main() -> mongodb::error::Result<()> {
+             // Get a handle to the cluster
+             let client = Client::with_uri_str(
+                 "mongodb+srv://<username>:<password>@<cluster-url>/test?w=majority",
+             )?;
+
+             client
+                 .database("admin")
+                 .run_command(doc! {"ping": 1}, None)?;
+             println!("Connected successfully.");
+
+             // List the names of the databases in that cluster
+             for db_name in client.list_database_names(None, None)? {
+                 println!("{}", db_name);
+             }
+             Ok(())
+         }
+
+   .. tab:: Async
+      :tabid: rust-async
+
+      Make sure you specified your async runtime. See
```

Comment:
We also think most users will be using the `tokio` runtime, which is the default and requires no configuration. Perhaps we could reword this to the something like the following:
> The default async runtime used by the driver is [`tokio`](tokio.rs). To use a different runtime, see [Configuring the async runtime](https://github.com/mongodb/mongo-rust-driver#configuring-the-async-runtime).